web/beacon-app/src/features/organizations/components/DeleteOrgModal/DeleteOrgModal.tsx

```diff
@@ -0,0 +1,29 @@
+import { Modal } from '@rotational/beacon-core';
+import { Fragment } from 'react';
+
+import { Close } from '@/components/icons/close';
+
+export default function DeleteOrgModal(props: any) {
+  return (
+    <div>
+      <Modal
+        title="Delete Organization"
+        open={true}
+        containerClassName="max-w-md"
+      >
+        <Fragment key=".0">
```

Comment:
Where this fragment key came from?

Reply:
It's listed in Storybook for the modal component. I kept getting an error message without it, so I added it. Below is a screenshot from the Storybook docs.

![Screenshot 2023-02-09 at 10 20 07 AM (2)](https://user-images.githubusercontent.com/94616884/217854847-3ae2d13d-86a6-453c-b6c0-8943d1e6dd4c.png)